views/layouts/main.handlebars

```diff
@@ -40,8 +40,8 @@
                 <div class="order-md-2 col-md-6">
                   <form id="search-form" action="" method="get" action="#">
                     <div class="input-group">
-                      <input type="text" class="form-control" autofocus="autofocus" name="username" placeholder="Github username" required>
-                      <button type="submit" class="input-group-addon btn" name=""><i class="fa fa-search"></i> &nbsp;Search</button>
+                      <input type="text" class="form-control" autofocus="autofocus" id="username" name="username" placeholder="Github username" required>
+                      <button type="submit" class="input-group-addon btn" name="" onClick="test()"><i class="fa fa-search"></i> &nbsp;Search</button>
```

Comment:
Also this, it would be good to name function based on its functionality so other contributors will understand, rather than `test()`